Tests iOS/BannerViewTests.swift

```diff
@@ -0,0 +1,221 @@
+import XCTest
+import Sails
+import NIO
+
+
+class BannerViewTests: XCTestCase {
```

Comment:
First off, I love these tests! So happy to see that setting the pasteboard content is so simple.

That said, I think it's a bit overkill to have a separate pair of tests for each tab. I get that we want to make sure that the banner appears (or doesn't appear) on each tab, but we need to way the cost of the tests against the value they provide.

The "cost" of the tests in this case is the time it takes for them to run. UI tests that require launching the app take quite a while to run (relative to unit tests that are typically very fast). 

I think we could reduce this cost if we simplify down to three tests:
1. save from clipboard banner appears on the home tab when foregrounding the app w/ a URL in the pasteboard
2. same as 1 but with a URL that has already been saved (assert the item is moved to top of list) 
3. save from clipboard banner _does not appear_ when foregrounding the app without a URL in the pasteboard

In tests 1 and 3, you could switch between each tab to make sure the banner is still present (or not as the case may be).

If we organize the tests in that way, then I think we can get the same general value but at a much lower cost of running. What do you think?

Reply:
I agree! What do we think about keeping the dismissing of the banner as test 4?

Reply:
oh, right. Yes let's keep that one as well